auditlog_security/views/auditlog_view.xml

```diff
@@ -0,0 +1,119 @@
+    </record>
+
+    <!-- show logs for partner out side of technical too, now that we have rules
+    for all types of users, will place in "contact" -->
+    
+
+    <record id="auditlog_form_view_ext" model="ir.ui.view">                                       
+        <field name="name">auditlog rule form extension</field>                   
+        <field name="model">auditlog.rule</field>                                             
+        <field name="inherit_id" ref="auditlog.view_auditlog_rule_form"/>                         
+        <field name="arch" type="xml">                                                  
+            <xpath expr="//group[last()]" position="after">         
+                <group>
+                    <!-- in the domain are the blacklisted fields in auditlog they 
+                        would be skipped anyway, so we repeat here to avoid incongruencies 
+                    -->
+                    <div><p>
+                        Add fields here to make any changes to them (audit log lines) 
+                        visible to members of the selected groups.
+                    </p></div>
+
+                    <field name="auditlog_line_access_rule_ids">
+                        <tree delete="1" create="1" edit="0">
+                            <field name="name" readonly="1"/>
+                            <field name="model_id" readonly="1" invisible="1"/>
+                            <field name="group_ids"                          
+                                   options="{'no_create': True}"
+                             />
+                            <field name="field_ids"                           
+                                    options="{'no_create': True}"
+                            />
+                        </tree>
+                        <form >
+                            <sheet>
+                                <group>
+                                    <field name="model_id" readonly="1" invisible="1" />
+                                    <field name="auditlog_rule_id" readonly="1" invisible="1"/>
+                                    <field name="group_ids" 
+                                        attrs="{'readonly': [('state', '=', 'subscribed')]}"
+                                        >                      
+                                    <tree editable="bottom">
+                                        <field name="name" readonly="1"/>
+                                        <field name="full_name" readonly="1"/>
+                                    </tree>
+
+                                </field>
+                                    <field name="field_ids"
+                                        domain="[('model_id', '=' , model_id),('name', 'not in', ['id', 'create_uid', 'create_date', 'write_uid', 'write_date','display_name', '__last_update']) ]"
+                                        attrs="{'readonly': [('state', '=', 'subscribed')]}"
```

Comment:
These two references to `state` field are giving issues:

```
Field 'state' used in attributes must be present in view but is missing:
 - 'state' in attrs="{'readonly': [('state', '=', 'subscribed')]}"
 - 'state' in attrs="{'readonly': [('state', '=', 'subscribed')]}"
```

Not seeing `state` field in the model of `auditlog.rule` anymore, it was moved to the request object, so I guess this can be removed?

Reply:
double checking.

Reply:
@thomaspaulb  solved by https://github.com/sunflowerit/server-tools/pull/4/commits/4cabfa5aae4a6bb4319e1bbb63efffe1f6bf0b11 
Untested, I can test after I'm done with a current debug session.